src/components/shared/profileCard/index.js

```diff
@@ -10,49 +10,80 @@ import React from "react";
 import "./profileCard.scss";
 import { ReactComponent as PhoneIcon } from "../../../assets/svgs/mobile.svg";
 import { ReactComponent as MailIcon } from "../../../assets/svgs/mail.svg";
-const ProfileCard = () => {
+import { getAvatarName } from "../../../helpers/functions";
+import { object } from "prop-types";
+const ProfileCard = ({ content }) => {
     return (
         <Card className="profileCardWrap" elevation={1}>
             <CardActionArea className="actionWrap">
-                <Avatar
-                    alt="Remy Sharp"
-                    src="/logo512.png"
-                    className="avatarMedia"
-                />
+                {content?.avatarUrl ? (
```

Comment:
React material Avator take alt attribute as name,so if there is not image it can still show avatar base on name.

Reply:
I already tried with default alt but it's only shows 1 chars so that i used that approach